docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>List alert activations for a given time range.</p><p>To filter, pass&nbsp;<code>since</code>&nbsp;and&nbsp;<code>until</code>&nbsp;as&nbsp;<code>datatime.datatime</code>&nbsp;objects:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>